Instagram/Views/CommentCell.m

```diff
@@ -0,0 +1,22 @@
+//
+//  CommentCell.m
+//  Instagram
+//
+//  Created by Megan Miller on 6/28/22.
+//
+
+#import "CommentCell.h"
+#import <Parse/Parse.h>
+
+@implementation CommentCell
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+}
+
+- (void)refreshData {
+    [self.comment fetchIfNeeded];
```

Comment:
Generally you shouldn't put network or data logic in views. A view's sole purpose is to take data and put it onto the screen. Any data manipulation should be handled elsewhere.